problemtools/verifyproblem.py

```diff
@@ -170,18 +222,24 @@ def run_submission(self, sub, args, timelim_low=1000, timelim_high=1000):
         if res2.runtime <= timelim_low:
             res1 = res2
         else:
-            res1 = SubmissionResult('TLE', score=self.testcasegroup.config['reject_score'])
-        res1.testcase = res2.testcase = self
-        res1.runtime_testcase = res2.runtime_testcase = self
+            res1 = SubmissionResult('TLE')
         res1.runtime = res2.runtime
-        if res1.verdict == 'AC':
-            res1.ac_runtime = res1.runtime
-            res1.ac_runtime_testcase = res1.runtime_testcase
-        if res2.verdict == 'AC':
-            res2.ac_runtime = res2.runtime
-            res2.ac_runtime_testcase = res2.runtime_testcase
-        self.info('Test file result: %s)' % (res1))
-        return (res1, res2)
+        self._result_cache = (cache_key, (res1, res2))
+        return (res1, res2, False)
+
+    def _apply_default_score(self, res):
```

Comment:
This function is a bit misnamed since it does more than just applying default score.  Maybe `_init_result_defaults`?